lib/node/fullnode.js

```diff
@@ -646,6 +646,77 @@ class FullNode extends Node {
 
     return this.filterindex.getFilter(hash);
   }
+
+  /**
+   * Retrieve compact filter header by hash.
+   * @param {Hash | Number} hash
+   * @returns {Promise} - Returns {@link hash}.
+   */
+
+  async getBlockFilterHeader(hash) {
```

Comment:
Needs to accept filter Type as argument?

Reply:
will add this once #1057 is merged